website/docs/docs/collaborate/auto-exposures.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Auto-exposures"
+sidebar_label: "Auto-exposures"
+description: "Import and auto-generate exposures from dashboards and understand how models are used in downstream tools for a richer lineage."
+pagination_prev: null
+pagination_next:  "docs/collaborate/data-tile"
+---
+
+# Auto-exposures <Lifecycle status='beta' />
+
+As a data team, it’s critical that you have context into the downstream use cases and users of your data products. Auto-exposures integrates natively with Tableau (Power BI coming soon) and auto-generates downstream lineage in dbt Explorer for a richer experience.
```

Comment:
Hiya @mirnawong1 

As a non-blocking suggestion, would it be worthwhile linking to the Tableau doc: https://docs.getdbt.com/docs/cloud-integrations/semantic-layer/tableau?

Kind Regards

Reply:
good question! they're diff docs and not related. tableau is a bi tool so the SL x tableau integration is for SL metric querying.